src/components/experiment-tracking/runs-list-card/runs-list-card.js

```diff
@@ -52,7 +76,15 @@ const RunsListCard = ({
         <div className="runs-list-card__id">{id}</div>
         <div className="runs-list-card__timestamp">{humanReadableTime}</div>
       </div>
-      {bookmark && <BookmarkIcon className={'runs-list-card__bookmark'} />}
+      {bookmark ? (
+        <BookmarkIcon className={'runs-list-card__bookmark'} />
+      ) : (
+        <BookmarkStrokeIcon
+          className={
+            'runs-list-card__bookmark runs-list-card__bookmark--stroke'
+          }
+        />
+      )}
```

Comment:
please add a test for testing the expected display of the different icons

Reply:
Added.